pkg/codegen/schema/docs_test.go

```diff
@@ -123,23 +123,22 @@ func getDocsForPackage(pkg *Package) []doc {
 	return allDocs
 }
 
+//nolint:paralleltest // needs to set plugin acquisition env var
 func TestParseAndRenderDocs(t *testing.T) {
-	t.Parallel()
-
 	files, err := os.ReadDir(testdataPath)
 	if err != nil {
 		t.Fatalf("could not read test data: %v", err)
 	}
 
-	//nolint:paralleltest // false positive because range var isn't used directly in t.Run(name) arg
+	//nolint:paralleltest // needs to set plugin acquisition env var
 	for _, f := range files {
 		f := f
 		if filepath.Ext(f.Name()) != ".json" || strings.Contains(f.Name(), "awsx") {
 			continue
 		}
 
 		t.Run(f.Name(), func(t *testing.T) {
-			t.Parallel()
+			t.Setenv("PULUMI_DISABLE_AUTOMATIC_PLUGIN_ACQUISITION", "false")
```

Comment:
TIL about `t.Setenv`, I always used to manually reset the environment variable after the test :facepalm: